sdk/core/azure-core-tracing-opentelemetry/src/samples/PublishEventsTracingSample.md

```diff
@@ -23,7 +23,7 @@ Sample uses **[opentelemetry-sdk][opentelemetry_sdk]** for implementation and **
 <dependency>
     <groupId>com.azure</groupId>
     <artifactId>azure-messaging-eventhubs</artifactId>
-    <version>5.0.0-beta.6</version>
+    <version>5.0.0-beta.7</version>
```

Comment:
I think this should be marked as a dependency version and we should update how these samples are injected into the README to ensure they compile and use the right version.

Reply:
https://github.com/Azure/azure-sdk-for-java/issues/7070